dev/generate-r-files/data/class-names.tsv

```diff
@@ -0,0 +1,9 @@
+class_name	env_name
+polars_data_frame	polars_dataframe__methods
+polars_group_by	polars_groupby__methods
+polars_rolling_group_by	polars_rolling_groupby__methods
+polars_dtype	polars_datatype__methods
```

Comment:
Not specifically related to this PR but shouldn't this be named `polars_dtype_methods` for consistency?